### PR TITLE
M2m tags

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

File: anyblok_furetui/furetui/resource.py [hunk 1]
```diff
@@ -199,8 +199,13 @@ def get_field_for_x2Many(self, field, relation, description, fields2read):
...
         return self.get_field_for_x2Many(field, 'One2Many', description, [])
 
+    def get_field_for_Many2ManyTags(self, field, description, fields2read):
+        return self.get_field_for_Many2One(
+            field, description, fields2read, relation="Many2ManyTags")
+
     def get_field_for_Many2Many(self, field, description, fields2read):
-        return self.get_field_for_x2Many(field, 'Many2Many', description, [])
+        return self.get_field_for_Many2One(
+            field, description, fields2read, relation="Many2Many")
```
Comment (petrus-v) [inline anyblok_furetui/furetui/resource.py:208]: to revert

File: anyblok_furetui/furetui/resource.py [hunk 2]
```diff
@@ -627,6 +632,9 @@ def field_for_Many2One(self, field, fields2read, **kwargs):
     def field_for_One2One(self, field, fields2read, **kwargs):
         return self.field_for_relationship(field, fields2read, **kwargs)
 
+    def field_for_Many2ManyTags(self, field, fields2read, **kwargs):
+        return self.field_for_relationship(field, fields2read, **kwargs)
```
Comment (petrus-v) [inline anyblok_furetui/furetui/resource.py:636]: pas nécessaire

File: anyblok_furetui/furetui/resource.py [hunk 3]
```diff
@@ -739,8 +747,10 @@ def get_definitions(self, **kwargs):
             fields.sort()
             for field_name in fields:
                 field = fd[field_name]
-                if field['type'] in ('FakeColumn', 'Many2Many', 'One2Many',
-                                     'Function'):
+                if field['type'] in (
+                    'FakeColumn', 'Many2Tags', 'Many2Many', 'One2Many',
+                    'Function',
```
Comment (petrus-v) [inline anyblok_furetui/furetui/resource.py:752]: to revert !

File: anyblok_furetui/furetui/crud.py [hunk 4]
```diff
@@ -94,6 +94,7 @@ def read(cls, request):
             request.authenticated_userid
         )
         query = qs.from_filter_by(query)
+        query = qs.from_filter_by_primary_keys(query)
```
Comment (jssuzanne) [inline anyblok_furetui/furetui/crud.py:97]: add query = qs.from_composite_filter_by(query) where issue 64 of rest api will be merged

**Review by jssuzanne — CHANGES_REQUESTED** [review]

File: anyblok_furetui/furetui/crud.py (hunk 4 above)
Comment (petrus-v) [reply anyblok_furetui/furetui/crud.py:97]: +1

Comment (petrus-v) [reply anyblok_furetui/furetui/crud.py:97]: pushed

**Comment by jssuzanne** [comment]
Thanks for all

**Pull request merged**